test/3_records_test.js

```diff
@@ -382,4 +382,24 @@ contract("LinniaRecords", (accounts) => {
       assert.equal(tx3.logs[0].event, "LogRecordAdded")
       })
     })
+  describe("destructible", () => {
```

Comment:
i think following the convention, this should be named after an action (destroy). but its not a big deal

Reply:
also theres no test for destroy and send